swan-lake/integration-tutorials/transforming-message-formats.md

```diff
@@ -184,6 +188,9 @@ Follow the instructions given in this section to develop the service.
     };
     ```
 
+    > **Note:**
+    > Since `ssn` must definitely be an array of three integers, make it as a fixed-length [array](https://ballerina.io/learn/by-example/arrays/).
```

Comment:
```suggestion
    > Since `ssn` must definitely be an array of three integers, make it a fixed-length [array](https://ballerina.io/learn/by-example/arrays/).
```